src/openms/thirdparty/evergreen/src/Evergreen/evergreen.hpp

```diff
@@ -9,9 +9,23 @@
 #define EVERGREEN_ALWAYS_INLINE inline __attribute__((always_inline))
 #endif
 
-// for convenience:
-#include "../Utility/vector_ostream.hpp"
+// added by jpfeuffer to throw exceptions
+#include <stdexcept>
+#include <sstream>
 
+// added by jpfeuffer to enable header guards before the namespace evergreen
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+#include <iostream>
+#include <iomanip>
+#include <chrono>
+#include <array>
+#include <set>
+#include <cmath>
+#include <list>
+#include <map>
+#include <utility>
```

Comment:
this is fine as a hack,  but the more elegant solution is to add the `namespace evergreen {}` to every single `.h` file. Then it would also be possible to include individual `.h` file of the library and use additional STL includes in the library, but I guess this is fine for now.